kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -86,6 +87,18 @@ export default [
       store.commit('CORE_SET_ERROR', null);
     },
   },
+  {
+    name: PageNames.CUSTOM_TOPICS_CHANNEL,
+    path: '/topics/:channel_id',
```

Comment:
Having duplicate paths is an issue here - on my machine, this simply continues with the old behaviour.

I think it would be best to put the conditionality inside the main root channel component, and only have a single path here - this would also mean that you don't have to duplicate the handler above.

Reply:
Yeah, I think without this change, there might be problems when refreshing or going directly to the URL for a custom channel, since it's custom-ness isn't encoded in the URL anywhere.